frontend/src/app/firewood/forests/order-confirmation/order-confirmation.component.html

```diff
@@ -8,7 +8,7 @@ <h1 id="skip-nav-target" tabindex="-1" class="usa-heading">Firewood permit confi
 
     <div class="confirmation-details-container margin-top-2">
       <p>Your firewood permit has been purchased.</p>
-      <p>Your permit, load tags and required documents have been sent to <strong>email@address.com</strong>. If for some reason this email address is incorrect, please print your documents now.</p>
```

Comment:
There was a plan to clean this up in a follow up issue after we verify we are getting the `permit` object correctly on staging. It's not that it would be very complicated to merge this change into my branch when the time comes, but for the sake of scope creep can we please omit this change as well?